sys/Makefile

```
@@ -1,3 +1,8 @@
 MODULE = turpial_firmware_sys
 
+ifneq (,$(filter turpial_firmware_net,$(USEMODULE)))
+  DIRS += net
+  DIRS += storage
+endif
```

```suggestion
endif

ifneq (,$(filter turpial_firmware_storage,$(USEMODULE)))
  DIRS += storage
endif
```

Add other `ifneq` block for storage module.